openmetadata-ui/src/main/resources/ui/src/components/common/Card/CardV1.tsx

```diff
@@ -13,19 +13,24 @@
 
 import { Card } from 'antd';
 import { lowerCase } from 'lodash';
-import React from 'react';
-
+import React, { CSSProperties } from 'react';
 interface CardProps {
-  description: string;
+  children: React.ReactElement | string;
```

Comment:
We can extend this interface with HTMLAttributes so we don't have to define the `children`, `style` and `className` props explicitly.

```suggestion
interface CardProps extends HTMLAttributes<HTMLDivElement> {
  id: string;
  heading?: string;
}
```